packages/models/test/fixtures/eslint-plugin.mock.ts

```diff
@@ -31,7 +31,7 @@ export function eslintPluginConfig(outputDir = 'tmp'): PluginConfig {
     ...eslintMeta,
     runner: runnerConfig(
       Object.values(ESLINT_AUDITS_MAP),
-      join(outputDir, 'eslint-out.json'),
+      join(outputDir, `eslint-out.json`),
```

Comment:
Unnecessary change:

```suggestion
      join(outputDir, 'eslint-out.json'),
```

Reply:
The project can enable the eslint rule [quotes](https://typescript-eslint.io/rules/quotes/) to reduce unnecessary template strings.